### ISSUE TITLE
ValueError: too many values to unpack CMSPLUGIN_CASCADE

### ISSUE THREAD

**Issue by d9tecnologies** [awesto/cookiecutter-django-shop#12]
I have followed all the steps indicated in https://github.com/awesto/cookiecutter-django-shop to install django-shop but it gives me the error "too many values ​​to unpack in app_settings.CMSPLUGIN_CASCADE". 

Thanks for job!

**Comment by jrief** [comment]
Could you please dumpt the whole error report. Use http://dpaste.de if you want. Thanks

**Comment by d9tecnologies** [comment]
Yes, when I excecute "pipenv run ./manage.py initialize_shop_demo":

https://dpaste.de/FqvK

If I comment 491 line ("'shop.cascade.plugin_base.CatalogLinkForm'") of settings.py the error disappears, but then this:

https://dpaste.de/nwva

Thanks!
